site/src/patterns/hero.md

```diff
@@ -53,11 +53,11 @@ Hero with a theme-aware companion image.
 
 ## Accent hero with details
 
-This component variant, known as 'Accent Hero with Details', overlays an element on top of the hero's image.
+This component variant, known as 'Accent Hero with Details', overlays an element on top of the hero's image. `.hero-details` will require the use of `.flex-direction-row-tablet` to be applied to the `.hero` markup.
```

Comment:
I might suggest a rewording here, for clarity, since at first blush I thought this meant the `.flex-direction-row-tablet` needed to be applied to `.hero-details`.

```suggestion
This component variant, known as 'Accent Hero with Details', overlays an element on top of the hero's image. Note that to place the details on the image correctly across screen sizes, the hero component should also have the `.flex-direction-row-tablet` atomic.
```